### DISCUSSION TITLE
What are the commands to compile libgc.a manually?

### DISCUSSION THREAD

**Discussion by przemyslaw0** [vlang/v#19107] [Questions and Answers]
When I convert a .v file to .c, for compiling manually, there is a library for garbage collection called libgc.a.

But I cannot find any tip on how to compile thirdparty/tcc/lib/libgc.a manually.

How can a I compile libgc.a manually?

**Comment by JalonSolov** [comment]
`thirdparty/tcc/lib/libgc_build_cmd.txt`

**Reply by przemyslaw0** [reply]
I know this text file, but there is no ./configure file in thirdparty/libgc or thirdparty/tcc/*

**Reply by jlmess77** [reply]
Take a look at the GNUmakefile, it shows how it's built.
